ports/zephyr/main.c

```diff
@@ -132,16 +136,26 @@ void gc_collect(void) {
     //gc_dump_info();
 }
 
+#if !MICROPY_READER_VFS
 mp_lexer_t *mp_lexer_new_from_file(const char *filename) {
     mp_raise_OSError(ENOENT);
 }
+#endif
 
 mp_import_stat_t mp_import_stat(const char *path) {
+#if MICROPY_VFS
```

Comment:
code style: we try to indent preprocessor directives to the level of the code it protects, so here it would have one level of indentation (4 spaces)